docs/rules/comma-dangle.md

```diff
@@ -12,6 +12,26 @@ var foo = {
 ```
 
 On the other hand, trailing commas simplify adding and removing items to objects and arrays, since only the lines you are modifying must be touched.
+Additionally, another argument in favor of trailing commas is that it improves the clarity of diffs when an item is added or removed from an object or array:
```

Comment:
Because the original sentence implies improved clarity of diffs, let’s rewrite it to make that benefit explicit with just one sentence. Something like:

If object and array literals have trailing commas, adding or removing items at the end does not require a change to the adjacent line, which improves the clarity of diffs.